app/src/components/SearchComponent.tsx

```diff
@@ -20,6 +21,8 @@ interface Parameter {
  * SearchComponent is a component for displaying search inputs for a resource
  */
 export default function SearchComponent({ resourceType }: SearchComponentProps) {
+  const publicUrl = trpc.service.getPublicUrl.useQuery();
```

Comment:
I noticed that this was removed from `ReleaseModal.tsx`, was that purposeful?

Reply:
After your child artifacts PR the font end stopped interacting directly with the service. So after my rebase it was no longer needed.